### ISSUE TITLE
Trivial errors in RP2040 datasheet

### ISSUE THREAD

**Issue by arg08** [raspberrypi/pico-feedback#246]
(based on version build-date: 2021-11-04 build-version: 150df05-clean)

2.4.8, M0PLUS: CCR Register (P87 of the PDF): two of the fields are documented as "always reads as 1" but the reset value is listed as 0x0

M0PLUS: CCR Register (P98 of the PDF): the CAUTION box at the end of the section has an incomplete sentence saying "Starting the channel".  I'm not quite sure what that is trying to say - the first sentence makes sense on its own, so this was either intended to make a different point or is just junk from editing

**Comment by JamesH65** [comment]
I think the second of those is fixed in the latest release (actually today), but the first is still present/inconsistent,

**Comment by lurch** [comment]
Yes, the second of those (where it's talking about `CHAN_ABORT` on DMA channels, nothing to do with CCR?) has already been fixed in the latest release of the RP2040 datasheet (that section has been tweaked due to  RP2040-E13).

Pinging @nickfrancis55 regarding the first issue: the `STKALIGN` and `UNALIGN_TRP` fields in the M0PLUS `CCR` register both say "Always reads as one, ..." but that's not reflected in the Reset value of these RO fields.
![Screenshot from 2022-06-20 15-04-32](https://user-images.githubusercontent.com/476186/174619526-eb9de479-51be-40a9-b103-ba6d58e384e3.png)

**Comment by lurch** [comment]
@nathan-contino I just stumbled across an old email from Nick Francis where he says "Yep this should have reset value read as 1 for those 2 bits."
